packages/utilities/src/utilities/smoothScroll/smoothScroll.js

```diff
@@ -11,19 +11,27 @@
  * @example
  * import { smoothScroll } from '@carbon/ibmdotcom-utilities';
  *
- * <a href="#anchorlinkname" onClick={smoothScroll(e)}>lorem ipsum</a>
+ * Here e is event param and seletor is param where you want to apply smoothscroll
+ * <a href="#anchorlinkname" onClick={smoothScroll({ e, selector })}>lorem ipsum</a>
  *
  * You can use this for jump to target element by providing event object.
  * it will scroll into view of target by selecting attribute and assigning to id.
  *
- * @param {*} e event object
+ * @param {*} e event object and seletor
```

Comment:
Maybe adding a description like
`@param {object} {e: Event, Selector: QuerySelector for the target element}`

Reply:
done